mk/kafka.mk

```
@@ -1,95 +1,19 @@
 ZOOKEEPER_CLIENT_PORT ?= 2181
 # The list of topics to be created; if more than one split them by a space
 ifeq (,$(KAFKA_TOPICS))
 	$(warning KAFKA_TOPICS is empty; probably missed definition at mk/variables.mk)
 endif
-# KAFKA_TOPICS ?= platform.content-sources.introspect
-KAFKA_GROUP_ID ?= content-sources
-
-# The Kafka configuration directory that will be bound inside the containers
-KAFKA_CONFIG_DIR ?= $(PROJECT_DIR)/kafka/config
-# The Kafka data directory that will be bound inside the containers
-# It must belong to the repository base directory
-KAFKA_DATA_DIR ?= $(PROJECT_DIR)/kafka/data
-
-# KAFKA_BOOTSTRAP_SERVERS ?= localhost:9092,localhost:9093
-KAFKA_BOOTSTRAP_SERVERS ?= localhost:9092
-
-# https://kafka.apache.org/quickstart
-.PHONY: kafka-up
-kafka-up: DOCKER_IMAGE=$(KAFKA_IMAGE)
-kafka-up:  ## Start local kafka containers
-	[ -e "$(KAFKA_DATA_DIR)" ] || mkdir -p "$(KAFKA_DATA_DIR)"
-	$(DOCKER) container inspect kafka-zookeeper &> /dev/null || $(DOCKER) run \
-	  -d \
-	  --rm \
-	  --name kafka-zookeeper \
-	  -e ZOOKEEPER_CLIENT_PORT=$(ZOOKEEPER_CLIENT_PORT) \
-	  -e ZOOKEEPER_OPTS="$(ZOOKEEPER_OPTS)" \
-	  -v "$(KAFKA_DATA_DIR):/tmp/zookeeper:z" \
-	  -v "$(KAFKA_CONFIG_DIR):/tmp/config:z" \
-	  -p 8778:8778 \
-	  -p 9092:9092 \
-	  -p 2181:2181 \
-	  --health-cmd /opt/kafka/scripts/zookeeper-healthcheck.sh \
-	  --health-interval 5s \
-	  --health-retries 10 \
-	  --health-timeout 3s \
-	  --health-start-period 3s \
-	  "$(DOCKER_IMAGE)" \
-	  /opt/kafka/scripts/zookeeper-entrypoint.sh
-	$(DOCKER) container inspect kafka-broker &> /dev/null || $(DOCKER) run \
-	  -d \
-	  --rm \
-	  --name kafka-broker \
-	  --net container:kafka-zookeeper \
-	  -e KAFKA_BROKER_ID=1 \
-	  -e KAFKA_ZOOKEEPER_CONNECT="zookeeper:2181" \
-	  -e KAFKA_ADVERTISED_LISTENERS="PLAINTEXT://kafka-broker:9092" \
-	  -e ZOOKEEPER_CLIENT_PORT=$(ZOOKEEPER_CLIENT_PORT) \
-	  -e KAFKA_LISTENER_SECURITY_PROTOCOL_MAP=PLAINTEXT:PLAINTEXT \
-	  -e KAFKA_INTER_BROKER_LISTENER_NAME=PLAINTEXT \
-	  -e KAFKA_OPTS='-javaagent:/usr/jolokia/agents/jolokia-jvm.jar=host=0.0.0.0' \
-	  -e KAFKA_TOPICS="$(KAFKA_TOPICS)" \
-	  -v "$(KAFKA_DATA_DIR):/tmp/zookeeper:z" \
-	  -v "$(KAFKA_CONFIG_DIR):/tmp/config:z" \
-	  "$(DOCKER_IMAGE)" \
-	  /opt/kafka/scripts/kafka-entrypoint.sh
 
-.PHONY: kafka-down
-kafka-down: DOCKER_IMAGE=$(KAFKA_IMAGE)
-kafka-down:  ## Stop local kafka infra
-	! $(DOCKER) container inspect kafka-broker &> /dev/null || $(DOCKER) container stop kafka-broker
-	! $(DOCKER) container inspect kafka-zookeeper &> /dev/null || $(DOCKER) container stop kafka-zookeeper
-	$(DOCKER) container prune -f
-
-.PHONY: kafka-clean
-kafka-clean: kafka-down  ## Clean current local kafka infra
-	export TMP="$(KAFKA_DATA_DIR)"; [ "$${TMP#$(PROJECT_DIR)/}" != "$${TMP}" ] \
-	    || { echo "error:KAFKA_DATA_DIR should belong to $(PROJECT_DIR)"; exit 1; }
-	rm -rf "$(KAFKA_DATA_DIR)"
+KAFKA_CONTAINER_NAME=$(COMPOSE_PROJECT_NAME)_kafka_1
```

I had to update this line for docker in apple m1:
```suggestion
ifeq (podman,$(DOCKER))
KAFKA_CONTAINER_NAME=$(COMPOSE_PROJECT_NAME)_kafka_1
else
KAFKA_CONTAINER_NAME=$(COMPOSE_PROJECT_NAME)-kafka-1
endif
```